weblate/memory/forms.py

```diff
@@ -2,19 +2,58 @@
 #
 # SPDX-License-Identifier: GPL-3.0-or-later
 
+from pathlib import Path
+
 from django import forms
 from django.core.validators import FileExtensionValidator
 from django.utils.translation import gettext_lazy
 
+from weblate.lang.models import Language
+from weblate.utils.forms import SortedSelect
+
 
 class UploadForm(forms.Form):
     """Uploading file to a translation memory."""
 
     file = forms.FileField(
         label=gettext_lazy("File"),
-        validators=[FileExtensionValidator(allowed_extensions=["json", "tmx"])],
+        validators=[
+            FileExtensionValidator(allowed_extensions=["json", "tmx", "xliff"])
```

Comment:
Maybe allow more formats here?

Reply:
Is it fair to assume all bilingual formats listed in the [Supported file formats](https://docs.weblate.org/en/latest/formats.html) can be included, since some also support monolingual content ?
i.e `["json", "tmx", "xliff", "po", "ts", "properties", "strings", "csv"]`

Reply:
I'd definitely include at least `po` and skip all monolinguals, let's start with this:

```suggestion
            FileExtensionValidator(allowed_extensions=["json", "tmx", "xliff", "po", "csv"])
```